modular_skyrat/modules/xenos_skyrat_redo/code/base_skyrat_xeno.dm

```diff
@@ -0,0 +1,281 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat
+	name = "rare bugged alien"
+	icon = 'modular_skyrat/modules/xenos_skyrat_redo/icons/big_xenos.dmi'
+	rotate_on_lying = FALSE
+	base_pixel_x = -16 //All of the xeno sprites are 64x64, and we want them to be level with the tile they are on, much like oversized quirk users
+	/// Holds the ability for making an alien's sprite smaller to only themselves
+	var/datum/action/small_sprite/skyrat_xeno/small_sprite
+	/// Holds the ability for quick resting without using the ic panel, and without editing xeno huds
+	var/datum/action/cooldown/alien/skyrat/sleepytime/rest_button
+	/// Holds the ability for allowing a xeno to devolve back into a larve if they so choose
+	var/datum/action/cooldown/alien/skyrat/devolve/devolve_ability
```

Comment:
i'll just put it here, but devolve makes them far-far to hard to take out unless they get royally trapped (especially for a midround), because even if they're the wrong caste, they can just swap back to queen when she dies, which really hampers the flow especially if they do the run out into space/icebox thing over and over, and leaves them far to capable of never-being-able-to-be-killed-off.

i'd love to see more variance in our midrounds, but they need to be able to be handled promptly, because they all shouldn't be the star of the round when they show up.

Reply:
When the queen dies there is an EXTREMELY long cooldown before any other xeno can evolve into a queen, on top of a huge host of other effects that completely cripple xenos when the queen dies.

Reply:
> When the queen dies there is an EXTREMELY long cooldown before any other xeno can evolve into a queen, on top of a huge host of other effects that completely cripple xenos when the queen dies.
> 
> I also disagree with the logic that midrounds "need to be able to be promptly handled". Xenos are a very rare midround that, yes, like other midrounds, can snowball, but infrequently do so (on top of their spawn rate) and this just seems like you don't want midrounds ever doing well in a round, which makes playing them pointless and sad.

long cooldown or not, it removes the 'downside' to letting the queen die, and just putting them on a timer until they can do it all again. If you want to throw baseless accusations, I can just append a list of "You just want to play them, and win everytime" "You just want them to be overpowered so you can 'robust' powergamers" "You just want to abuse the fact they almost can't be killed off" But it's not fun to do that, they're a midround, and if you'll note the other midrounds, they tend to either be immobile(Blob, mold, vines), unable to truly 'take over' the round(borers, ninja), or don't self reproduce, and can be handled if you focus the 'big one' (space dragon), making one that can both be mobile, take over the round, and self-reproduce goes against everything a threat, that appears after the other threats, should be.